runexporterapp.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export IRONIC_CONFIG=/etc/ironic/ironic.conf
```

Comment:
I wonder if this is still needed

Reply:
It's necessary to have the variable set, not mandatory to be here.
The Flask app needs to know the `location` that is in the `ironic.conf` see [1]

[1] https://github.com/openstack/ironic-prometheus-exporter/blob/master/ironic_prometheus_exporter/app/exporter.py

Reply:
I would leave it here, just to be sure it's loaded by the exporter

Reply:
If so, we need to run configure-ironic, otherwise the file may not be present.